### ISSUE TITLE
Move NetCDF.jl to JuliaIO?

### ISSUE THREAD

**Issue by meggart** [JuliaIO/Roadmap#4]
I have a request to move https://github.com/JuliaGeo/NetCDF.jl to JuliaIO. The package is registered since 2013 and provides support for reading/writing NetCDF files and is hosted in the JuliaGeo org. There is an alternative package for reading NetCDF files https://github.com/Alexander-Barth/NCDatasets.jl which implements additional features (CF conventions) that are specific to geospatial data. 

Because people got confused by the two NetCDF packages there was the suggestion to move NetCDF.jl as a more general low-level package to JuliaIO and have NCDatasets in JuliaGeo so that users get a better impression on the package intention. See [this discussion](https://github.com/Alexander-Barth/NCDatasets.jl/issues/57)

**Comment by timholy** [comment]
Sounds reasonable to me. I assume you'd need someone to give you permissions to transfer it?

**Comment by meggart** [comment]
> I assume you'd need someone to give you permissions to transfer it?

Yes that is the case. However, I would prefer to wait for a couple of days. I linked this issue from the other discussion so just in case someone wants to object...

**Comment by timholy** [comment]
Sure, just ping this again when you're ready.

**Comment by alex-s-gardner** [comment]
@meggart was any decision/action made on this? It would be nice to have NetCDF.jl as part of JuliaIO and for NCDatasets.jl to move into JuliaGeo, hopefully building on top of NetCDF.jl to minimize maintenance. The current two package world makes it difficult to know what to use and how to contibute. It would be nice to bring a conclusion to https://github.com/Alexander-Barth/NCDatasets.jl/issues/57
